packages/graph/hash_graph/bin/hash_graph/src/args.rs

```diff
@@ -29,14 +29,14 @@ pub struct Args {
     /// - be in the standard format accepted by Rust's `regex` crate.
     ///
     /// - contain a capture group named "shortname" to identify a user's shortname, e.g.
-    ///   `(?P<shortname>[\w]*)`
+    ///   `(?P<shortname>[\w|-]+)`
     ///
     /// - contain a capture group named "kind" to identify the slug of the kind of ontology type
     ///   being hosted (data-type, property-type, entity-type, link-type), e.g.
     ///   `(?P<kind>(?:data-type)|(?:property-type)|(?:entity-type)|(?:link-type))`
     #[clap(
         long,
-        default_value_t = Regex::new(r"http://localhost:3000/@(?P<shortname>\w+)/types/(?P<kind>(?:data-type)|(?:property-type)|(?:entity-type)|(?:link-type))/(?:\w|-)+/").unwrap(),
+        default_value_t = Regex::new(r"http://localhost:3000/@(?P<shortname>[\w|-]+)/types/(?P<kind>(?:data-type)|(?:property-type)|(?:entity-type)|(?:link-type))/(?:\w|-)+/").unwrap(),
```

Comment:
I don't think this makes a big difference as the workspace would catch the invalid cases, but there's a check for not having `-` as the first symbol as well 
https://github.com/hashintel/hash/blob/1f85bd20b795939e7e1319d46a487d7a532844cc/packages/hash/api/src/model/knowledge/account.fields.ts#L46

And I assume `\w` includes underscores too, right?

Reply:
It's a good catch but I'm actively not trying to validate a shortname here, just to be able to capture an existing one.

We won't be creating shortnames based on these URLs, so I'm going to leave as is :)